package.json

```diff
@@ -3,9 +3,12 @@
   "version": "0.1.3",
   "license": "MIT",
   "scripts": {
-    "lint": "eslint . --ext .ts --fix",
+    "lint": "yarn lint:nofix --fix",
+    "lint:nofix": "eslint . --ext .ts",
     "dev": "nodemon src/index.ts",
-    "start": "ts-node src/index.ts"
+    "start": "ts-node src/index.ts",
```

Comment:
can we add here `--transpile-only`?

Reply:
This is a tests-focused pr. You could open another PR for that